### ISSUE TITLE
[Documentation] Create Contributors.md file

### ISSUE THREAD

**Issue by Kajol-Kumari** [anitab-org/vms#1028]
## Description
Currently we don't any separate file in the repo showing the list of contributors.

### Update
- [ ] Create `Contributors.md` file that will contain all the basic information of all the contributors of this repo. along with their images.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

**Comment by atharwa-24** [comment]
I would like to work on it

**Comment by BALaka-18** [comment]
Hi @Kajol-Kumari, I am a participant of Open Source Hack (OSH). Can I work on this issue ?

**Comment by Sommie09** [comment]
I am participating in OSH, could I work on this?

**Comment by Kajol-Kumari** [comment]
@atharwa-24 @BALaka-18 @Sommie09 It's good to see you all interested in taking up this issue. But all the issues with the label of `open source hack` will be assigned to the OSH participants starting from 1st of september on the basis of `First come first serve` so please wait patiently till 1st of september and till then you can set up the repo. locally and get acquainted with the project functionalities and its codebase.

**Comment by BALaka-18** [comment]
> @atharwa-24 @BALaka-18 @Sommie09 It's good to see you all interested in taking up this issue. But all the issues with the label of `open source hack` will be assigned to the OSH participants starting from 1st of september on the basis of `First come first serve` so please wait patiently till 1st of september and till then you can set up the repo. locally and get acquainted with the project functionalities and its codebase.

Okay @Kajol-Kumari

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari can you pls explain that if we had to only made file or do something else also
can you please elaborate
thanks

**Comment by swaraj961** [comment]
Hey Can you assign me this issue on 1st September when this hack began thanks looking forward to work on it.

**Comment by Kajol-Kumari** [comment]
Hey @swaraj961 @vaibhavarora102  please understand that we are going to assign only one issue to a participant at a time and that only on the basis of `First come First Serve` so please do look for an issue with few or no claims. Also regarding the deliverable of this issue, assignee needs to create a file named `contributors.md` in the main directory and include all the basic details of the contributors taking reference from [here](https://github.com/anitab-org/vms/graphs/contributors). Please let me know if anyone needs any further clarification on this.

**Comment by nitish-awasthi** [comment]
Hi! @Kajol-Kumari  can I work on this issue for OSH?

**Comment by swaraj961** [comment]
> Hey @swaraj961 @vaibhavarora102 please understand that we are going to assign only one issue to a participant at a time and that only on the basis of `First come First Serve` so please do look for an issue with few or no claims. Also regarding the deliverable of this issue, assignee needs to create a file named `contributors.md` in the main directory and include all the basic details of the contributors taking reference from [here](https://github.com/anitab-org/vms/graphs/contributors). Please let me know if anyone needs any further clarification on this.

Okay sure

**Comment by SanketDG** [comment]
Since, we'd have to maintain this every time someone makes a code change, I'd prefer using something like https://allcontributors.org/ to automate this.

**Comment by Kajol-Kumari** [comment]
> Since, we'd have to maintain this every time someone makes a code change, I'd prefer using something like https://allcontributors.org/ to automate this.

cool! sounds good to me. So, let's integrate this to avoid manual change. Thanks @SanketDG for suggestion.

**Comment by vaibhavarora102** [comment]
so whether this issue will continue for Josh or will be closed due to this integration @Kajol-Kumari

**Comment by Kajol-Kumari** [comment]
> so whether this issue will continue for Josh or will be closed due to this integration @Kajol-Kumari

@vaibhavarora102 I have modifed the issue description. so there is no reason to close this. Now instead of manually creating the `contributor.md` asignee will search and integrate the required bot to do this job.

**Comment by vaibhavarora102** [comment]
Okay thanks 
Kindly assign me for this work
As I am participating in OSH and I will make pull request only after 1 Sept
Thanks@Kajol-Kumari

**Comment by Kajol-Kumari** [comment]
Hey Everybody, all who are interested in this issue. Please accept my apologies but according to the new guidelines we received, we will count `First come first serve` on the basis of comments/claims done 1st september onwards. So, everyone who are interested in this issue, do come again on 1st and comment first to get assigned.

**Comment by vaibhavarora102** [comment]
Hi @Kajol-Kumari , can you pls assign this issue to me as its already 1st of September and I guess OSH has started
Thanks

**Comment by Kajol-Kumari** [comment]
Hey @vaibhavarora102 please go ahead with this one :)

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari I had made initial commit can you pls check
till now contributing.md is made 
and is bot is installed but is not triggered for this readme but well be done once this merge request is approved
thanks
as i am following docs for same 
thanks

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari  is it ready for merging or any other step also needed

can you pls tell as I also want to work on other their maintainers are saying to first complete your previous one 
thanks

**Comment by vaibhavarora102** [comment]
hope now it will be merged soon

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari  kindly check 
I had squashed all the commits  and had updated everything instructed by you
thanks

**Comment by vaibhavarora102** [comment]
I am facing some problems in squashing and pushing back
but hopefully will solve issues by today's day end
thanks

**Comment by vaibhavarora102** [comment]
@Kajol-Kumari  I am done with squashing 
kindly check
and hope everything is correct know

**Comment by sujal111** [comment]
I am willing to work and solve this issue. Please assign me the same

**Comment by vaibhavarora102** [comment]
kindly check mine new PR
as discussed previously
